package.json

```diff
@@ -47,17 +47,17 @@
     "url": "https://github.com/ipfs/js-ipfs-api"
   },
   "devDependencies": {
-    "aegir": "^8.0.0",
+    "aegir": "^8.1.2",
     "chai": "^3.5.0",
     "gulp": "^3.9.1",
-    "hapi": "^15.0.2",
+    "hapi": "^15.2.0",
     "interface-ipfs-core": "^0.15.0",
     "ipfsd-ctl": "^0.16.0",
     "pre-commit": "^1.1.3",
-    "socket.io": "^1.4.8",
-    "socket.io-client": "^1.4.8",
+    "socket.io": "^1.5.1",
+    "socket.io-client": "^1.5.1",
```

Comment:
for the spawning of nodes when the requests come from the browser (IPFS Factory)